.travis.yml

```diff
@@ -0,0 +1,71 @@
+# This is a special configuration file to run tests on Travis-CI via
```

Comment:
just to double check, did we set up travis also for the master branch?

Reply:
Travis will build for all branches in which a .travis file exists.

Reply:
true, but it need environment variables set up and did we do this?

Reply:
Yes. All branches will use the same travis secrets/ environment variables.